src/EditorFeatures/CSharpTest/Diagnostics/GenerateType/GenerateTypeTests.cs

```diff
@@ -1571,7 +1571,7 @@ internal class T
 {
     public T(out DateTime d)
     {
-        d = default(DateTime);
+        d = default;
```

Comment:
Why make these changes? They don't seem related to the overall change here.

Reply:
These tests use the default language version, which used to be 7.0. When the default is changed to 7.3, the IDE takes advantage of the new "target-typed default" feature, which changes the generated code. This is the new expected baseline for this test.